lib/matplotlib/tests/test_ticker.py

```diff
@@ -1449,6 +1449,22 @@ def test_latex(self, is_latex, usetex, expected):
             assert fmt.format_pct(50, 100) == expected
 
 
+@pytest.mark.xfail(locale != "fy_DE.UTF-8",
```

Comment:
I don't understand this condition; `locale` is the imported module and can never equal a string.